ENVParser/Program.cs

```diff
@@ -1,54 +1,53 @@
 ﻿
 using ENVParser;
-using static ENVParser.DataDictionary;
+using ENVParser.Utils;
 internal class Program
 {
     private static void Main(string[] args)
     {
         if (args.Length == 0)
         {
             Console.WriteLine("\nUsage: ENVParser <file path> <csv|json>(optional)");
-            Console.WriteLine("\n\t - Please either drop an env file on the executable or provide an ENV file path as a command-line argument.");
-            Console.WriteLine("\t - Optionally, provide an output file type as the second argument (json or csv).");
+            Console.WriteLine("\n\t - Either drop an ENV or provide a file path as a command-line argument.");
+            //Console.WriteLine("\t - If proving an ENV file, you can optionally specify an output file type as the second argument.");
+            //Console.WriteLine("\t   JSON and CSV are supported for export (JSON is the default).");
             return;
         }
 
-        // Check ENV file
+        // Check the file exists
         string filePath = args[0];
         if (!File.Exists(filePath))
         {
             Console.WriteLine($"The specified file does not exist: '{filePath}'");
             return;
         }
-        if (!Path.GetExtension(filePath).Equals(".ENV", StringComparison.OrdinalIgnoreCase))
+
+        // Only accept ENV or ENV.json files
+        if (!Path.GetExtension(filePath).Equals(".ENV", StringComparison.OrdinalIgnoreCase) && !Path.GetExtension(filePath).Equals(".json", StringComparison.OrdinalIgnoreCase))
         {
-            Console.WriteLine($"The file must be an .ENV file: '{Path.GetExtension(filePath)}' provided");
+            Console.WriteLine($"The file must be an .ENV or an .ENV.json file: '{Path.GetExtension(filePath)}' provided");
             return;
         }
 
-        // Parse ENV File
-        byte[] envFile = File.ReadAllBytes(filePath);
+        EnvFile envFile = new();
+        
+        // Call the Read method to populate the envFile instance
+        using FileStream fileStream = new(filePath, FileMode.Open, FileAccess.Read);
+        using BigEndianBinaryReader reader = new(fileStream);        
+        envFile.Read(reader);
+        
+        // Write to JSON
+        string outputFile = Path.GetFullPath(filePath).ToString().Replace(".ENV", ".ENV.json");
+        JsonExporter exporter = new();
+        exporter.Export(outputFile, envFile);
+        Console.WriteLine($"\n\tJSON file created at: '{outputFile}'");
+        return;
 
-        // Set up Data Dictionary            
-        List<DataDictionaryEntry> entries = DataDictionary.LoadDataDictionary();
+        //using var stream = new FileStream($"{filePath}.bin", FileMode.Create, FileAccess.Write);
```

Comment:
Do you want to leave these comments in?

Reply:
For now yes, as I update them later on in a further pull request :)